### ISSUE TITLE
1.0.0 is broken on nuxt 2 without nuxt bridge

### ISSUE THREAD

**Issue by benavern** [mvrlin/nuxt-viewport#13]
Hello,
After the release of the 1.0.0 version I can't use the module.

It generates mjs files in the `runtime` directory (before there were js files)

Here is the error output:

```
vuejs-bpr        | > nuxt
vuejs-bpr        | 
vuejs-bpr        | 
vuejs-bpr        |  FATAL  Cannot find module './runtime/viewport'
vuejs-bpr        | Require stack:
vuejs-bpr        | - /data/bpr/node_modules/nuxt-viewport/lib/index.js
vuejs-bpr        | - /data/bpr/node_modules/@nuxt/core/dist/core.js
vuejs-bpr        | 
vuejs-bpr        |   Require stack:
vuejs-bpr        |   - node_modules/nuxt-viewport/lib/index.js
vuejs-bpr        |   - node_modules/@nuxt/core/dist/core.js
vuejs-bpr        |   at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
vuejs-bpr        |   at Function.resolve (node:internal/modules/cjs/helpers:108:19)
vuejs-bpr        |   at ModuleContainer.nuxtModule (node_modules/nuxt-viewport/lib/index.js:40:45)
vuejs-bpr        |   at ModuleContainer.addModule (node_modules/@nuxt/core/dist/core.js:239:34)
vuejs-bpr        |   at node_modules/@nuxt/utils/dist/utils.js:639:43
vuejs-bpr        |   at async ModuleContainer.ready (node_modules/@nuxt/core/dist/core.js:55:5)
vuejs-bpr        |   at async Nuxt._init (node_modules/@nuxt/core/dist/core.js:478:5)
vuejs-bpr        | 
vuejs-bpr        | 
vuejs-bpr        |    ╭─────────────────────────────────────────────────────────╮
vuejs-bpr        |    │                                                         │
vuejs-bpr        |    │   ✖ Nuxt Fatal Error                                    │
vuejs-bpr        |    │                                                         │
vuejs-bpr        |    │   Error: Cannot find module './runtime/viewport'        │
vuejs-bpr        |    │   Require stack:                                        │
vuejs-bpr        |    │   - /data/bpr/node_modules/nuxt-viewport/lib/index.js   │
vuejs-bpr        |    │   - /data/bpr/node_modules/@nuxt/core/dist/core.js      │
vuejs-bpr        |    │                                                         │
vuejs-bpr        |    ╰─────────────────────────────────────────────────────────╯

```

Is there something to do to make it work ? (I use nuxt 2 without nuxt-bridge)

**Comment by mvrlin** [comment]
Hello, that’s interesting, because the bridge is only dev dependency, so it should not be included in the build🤔I’ll take a look.

**Comment by mvrlin** [comment]
@benavern released 1.0.1, the bug was with the `mkdist` package, which generated `.mjs` files instead of `.js` 🕶

**Comment by benavern** [comment]
Great, this works!!! thank you :)
